lib/http/endpoint.js

```diff
@@ -241,7 +241,11 @@ const defaultEndpoint = endpointBase({
 const htmlEndpoint = endpointBase({
   resultWriter: (result, request, response) => {
     response.type('text/html');
-    response.send(frontendPage(result));
+    if (typeof result === 'string') {
+      response.send(result);
+    } else {
+      response.send(frontendPage(result));
```

Comment:
How else is `frontendPage()` used? I think the goal should be to remove `frontendPage()` entirely.